docs/openapi.json

```diff
@@ -537,7 +637,8 @@
             "description": "Required. New status.",
             "enum": [
               "start",
-              "stop"
+              "stop",
+              "deleted"
```

Comment:
I understand, but I just realised now, it was just trying to avoid having to wait for a new set of PRs to be integrated for a string change, but I will create then now no problem.